pkg/utils/parse_arf_result_test.go

```diff
@@ -882,4 +882,73 @@ Server 3.fedora.pool.ntp.org`
 
 		})
 	})
+
+	Describe("Testing for correct content parsing", func() {
+		Describe("Testing new content used", func() {
+			Context("Valid XCCDF", func() {
+				dsFilename = os.Getenv("DEFAULT_CONTENT_DS_FILE_PATH")
```

Comment:
What if we built all the data streams and put them in a test directory, then had the test iterate the directory and test each of them? My guess is that would be quicker than spinning up a new test runner for each data stream?